.github/workflows/ci.yml

```diff
@@ -1156,9 +1161,6 @@ jobs:
       continue-on-error: true
```

Comment:
@maximusron Maybe worth removing this line as part of this PR, so its clearer when this PR is a sucess without consulting the ci output.

Reply:
This will probably error out very fast, even if everything runs smoothly because the failing cppyy tests output std::error's to the console.

Reply:
Ok, let's do that in a separate PR then..